server/datastore/psql/all_maintenance_requests.go

```diff
@@ -1,12 +1,13 @@
 package psql
 
-import(
-	// Import the necessary packages
+import (
+	"fmt"
+    "github.com/a2fumn2022/gpmn-nsl/server/datastore"
 )
 
 /* AllMaintenanceRequests performs a SQL query to fetch all rows in the
 *  maintenance_requests table */
-func (db *DB) AllMaintenanceRequests() {
+func (db *DB) AllMaintenanceRequests() error {
```

Comment:
Since @seelthao1 is implementing this, you should remove this file from your branch. We can combine the two later.